### DISCUSSION TITLE
Modulo de html

### DISCUSSION THREAD

**Discussion by eduardocastro23** [Launch-X-Latam/MisionFrontEnd#156] [General]
Alguien se dio cuenta que en el módulo de HTML falta la explicación de los últimos dos temas? Y no viene la practica!!!

**Comment by doguedogue** [comment]
Tienes razón, y es que este es un tutorial super rápido, si quieres profundizar te aconsejo leer más material o ver algunos videos.

Recuerda que de HTML, CSS, JS, etc, hay libros y muchos recursos aqui en la web. Te comparto uno de mis favoritos: https://www.w3schools.com/html/

**Comment by VictorFalconi** [comment]
Puedes estudiar mas de html, css, javascript y otras tecnologias en https://www.freecodecamp.org Esta en ingles pero vale la pena. No se que tan bien funcione el traductor en esa pagina

**Reply by iam-leo** [reply]
[Freecodecamp](https://www.freecodecamp.org/) ya tiene su versión en español (desconozco si en su totalidad o solo las primeras lecciones) 
Solo hay que ir a Menu :arrow_right: Change language :arrow_right: Español

**Comment by maxkario** [comment]
Si, pero quizas se cambien algunas cosas cuando lleguemos a esa parte. Hay que ir con calma.
